index.html

```diff
@@ -241,6 +241,57 @@ <h3>Skills</h3>
 
     </section>
 
+    <!----------------- form area -->\
+    <section id="form-area1">  
+       
+        <div class="txt">
+            <P class="form-txt">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </P>
+        </div>
+
+              <form class="form-area" action="https://formspree.io/f/mjvdwrry" method="POST"> 
+                <form action="https://formspree.io/f/{form_id}" method="post">
+                <div class="user-details">
+                    
+                 
+                            <input type="text" placeholder="First name" required>
+                       
+                            <input type="email" placeholder="Email address"  pattern=".+@globex\.com" size="30" required >
+    
```

Comment:
- [ ] As requested by the first reviewer, kindly remove the `pattern` attribute you are using for this form element. All emails might not follow that pattern and that means users who have emails that do not follow this pattern would not be able to submit their request. Kindly change it. 👍 😎

Reply:
hi [Gambit142](https://github.com/Gambit142)
it resolved now